Library/Formula/ack.rb

```diff
@@ -1,4 +1,5 @@
 class Ack < Formula
+  desc "A search tool like grep, but optimized for programmers"
   homepage "http://beyondgrep.com/"
```

Comment:
> As far as I know, brew does not enforce any rules about the order of items in the top of the class.

This isn't quite true. It's not in the `audit` but the only things that move around in the initial block unless there's need for `stable do` is the `version` line, if there is one. That can go either below the URL or below the `sha`. But generally, although not audit-enforced, deviation from that tends to get asked to change. Whether that's just because formulae become a pain to update/compare/etc if all of them have different layouts or whether that's for another reason, I defer to the people who wrote the implementation.